hack/spin-local-kind.sh

```diff
@@ -1,11 +1,11 @@
 #!/bin/bash -e
 
 REGISTRY=${REGISTRY:=localhost:5000}
-IMAGE_TAG=${IMAGE_TAG:=0.9}
+IMAGE_TAG=${IMAGE_TAG:=latest}
```

Comment:
What is the purpose of defaulting to `latest` here?

Reply:
It would make more sense to default to "dev" tag, but the release script is overriding this with the release tag and during development the makefile replaces this with a sha, so it doesn't make much difference.